source/index.html.md

```diff
@@ -227,6 +227,12 @@ Tasks objects have a metadata parameter. You can use this parameter to attach ke
 
 Metadata is useful for storing additional, structured information on an object. As an example, you could store a video's unique identifier in your system on its content moderation `categorization` task. You can also use it to denote the end use case for the task, as "content moderation" or "data categorization" for example. Metadata is not used by Scale (e.g., to affect how the task is done).
 
+## Attachments
+
+Tasks often need a file associated with them that Scale API calls an attachment. For example, an annotation task requires an image file to show to a Scaler. These attachments are not limited to these formats and may be image, audio, video, or pdf files, or a website, or even plain text. For attachment types except plain text, the attachment will need to be specified via a URL. See the sections for specific tasks below for examples.
+
+Scale API will fetch the attachment via the URL you provide. If the URL does not return a 200, we will retry up to five times. If we still have not received a successful 200 response, the task will automatically be marked as completed and we will call your `callback_url` with an error message detailing the HTTP statusCodes and the attachment URL(s) that failed.
```

Comment:
I think this could be less specific here, and just say something like "If we cannot fetch your attachment URL, or the attachment is invalid (for instance, not a valid image for an image attachment), we will send your `callback_url` an error callback detailing the issue." That makes this more flexible as we change our policy, and also covers the bad image and bad audio file cases we're going to add soon.